Directory.Build.props

```diff
@@ -14,9 +14,6 @@
     <RepositoryRoot>$(MSBuildThisFileDirectory)</RepositoryRoot>
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)build\Key.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
-    <AssemblySigningCertName>Microsoft</AssemblySigningCertName>
-    <PackageSigningCertName>MicrosoftNuGet</PackageSigningCertName>
-    <PublicSign Condition="'$(OS)' != 'Windows_NT'">true</PublicSign>
```

Comment:
Is this because supports regular signing?

Reply:
PublicSign is actually a setting for strong name signing, not authenticode. I discovered the new sign tool normalizes public SN signing to full SN signing anyways. PublicSign is a left over from project.json conversion which doesn't need to be set anymore because roslyn supports full signing x-plat now (it didn't in .NET Core 1)